src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -231,4 +264,49 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			}
 		}
 	}
+
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered.
+	 * Returns markup created after user enters values and submits the callout.
+	 */
+	private async createCallout(type: MarkdownButtonType, triggerElement: HTMLElement): Promise<ILinkCalloutDialogOptions> {
```

Comment:
I didn't want the text transformer itself to have to know anything about callouts, so that's why I refactored it out.